.gitignore

```diff
@@ -9,3 +9,15 @@ __pycache__/
 
 # Ignore auto-saves
 *~
+
+# Ignore files under home/gold/testname which are not needed
+update
+score
+set
+rule
+phenomenon
+parameter
+output
+item-phenomenon
+fold
+analysis
```

Comment:
Aside from the missing newline after `analysis`, these are probably fine. They ignore files named with these patterns in *any* subdirectory. If we want to restrict to the directory mentioned in the comment, we could do `tests/regression/home/gold/*/update`, etc.